source/includes/php-mongodb-compatibility-table.rst

```diff
@@ -13,7 +14,19 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - ext 1.10 + lib 1.9
+     - |checkmark|
```

Comment:
IIRC, previous versions of these server compatibility tables had footnotes when driver releases were not fully compatible. In this case, PHPC 1.10 and PHPLIB 1.9 will be missing support for snapshot reads and load balancers.

AFAIK, load balancers are not specifically tied to server 5.0, but should we have a footnote about lacking support for snapshot reads? The plan for any drivers missing features is to release what we have on Tuesday, July 13 and follow up with a subsequent minor release ASAP with the remaining work.

Also: I'm only aware of the case for PHP, but it's possible another driver or two is in a similar boat (with possibly different features).

cc: @alcaeus

Reply:
Hi Jeremy,

I added a footnote to the PHP compatibility table, see here: 

https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/php/#mongodb-compatibility

Does that impart the correct/enough information?

Reply:
The corresponding drivers ticket is [DRIVERS-815](ttps://jira.mongodb.org/browse/DRIVERS-815), where this is referred to as "Snapshot reads on secondaries"; however, I'm not in a position to say whether that is the correct customer-facing name. The 5.0 server release notes refer to [Snapshots](https://docs.mongodb.com/v5.0/release-notes/5.0/#snapshots), which I presume is the same feature.

I would suggest one of the following:

 * Refer to "snapshot reads on secondaries"
 * Refer to "Snapshots" and link to the release notes for context

@terakilobyte: Can you weigh in here? @techbelle may also be in a better position to make the call on how best to refer to this.

Reply:
Staging links:

C: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/c/#footnote-c-1.18-driver-support

Go: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/go/#footnote-go-1.6-driver-support

PHP: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/php/#footnote-PHPC1.10-PHPLIB1.9-driver-support